app/scripts/modules/core/confirmationModal/confirm.html

```diff
@@ -43,6 +43,7 @@
       </div>
     </div>
   </div>
+  <task-comments ng-if="taskMonitor && params.submitMethod.length > 0" command="params"></task-comments>
```

Comment:
Since this modal is used by all the cloud providers, we should probably update all the cloud provider task modals to include comments, not just AWS.